ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml

```diff
@@ -1,186 +1,6 @@
-base_images:
```

Comment:
This deletion doesn't seem correct, it's probably related to my recent changes on release-next. I will take a look

Reply:
https://github.com/openshift-knative/eventing/pull/166

Reply:
Can you rerun it?

Reply:
I am now running